order/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+class OrderItem(models.Model):
+    product_id = models.ForeignKey('product.Product', on_delete=models.CASCADE)
```

Comment:
product_id로 column을 만들면 database table에 product_id_id로 생성이 되므로 product라고 하시면 됩니다!
```suggestion
    product = models.ForeignKey('product.Product', on_delete=models.CASCADE)
```

Reply:
네네, 바로 수정하겠습니다!!